assets/environment-indicator.css

```diff
@@ -7,25 +7,22 @@
 }
 
 #wpadminbar .altis-env-indicator {
-	background-color: #4667DE;
+	background-color: #4667de;
 	display: inline-block;
 	height: 1.2em;
+	line-height: 1.2;
 	margin-top: 6px;
 	padding: 2px 4px;
 	border-radius: 4px;
 	flex: 0;
 	word-break: normal;
 }
 
-#wpadminbar .altis-env-indicator {
-	line-height: 1.2;
-}
-
 #wpadminbar .altis-env-indicator abbr {
 	line-height: inherit;
 }
 
 #wpadminbar #wp-admin-bar-altis .dashicons-before::before {
-	font-size: 16px;
+	font-size: 1.2em;
```

Comment:
Our stylelint rules don't necessarily match the way the WP admin or the admin bar is styled so I'd make absolutely certain this is going to work consistently. I've generally opted for using stylelint ignore rules rather than changing the units.

Reply:
I tested this locally to make sure I got the sizing correct. This works, but if it's better to match the existing (bad) rules, I can ignore it in the stylelint rules.